### DISCUSSION TITLE
More games support?  (skyrim, Minetest,etc)

### DISCUSSION THREAD

**Discussion by 00pandora00** [modrinth/.github#189] [Feature requests]
**hi, sorry for been that guy, but i really need to know if there is any possibility for other games support?.**

I know that is not the platform's priority (and I don't expect it to be now). I just want to know if the team would be willing to have that option in the future. 

**Why am I worried about this?**

because I see that what happened with "forge web" can happen with "nexus mods" and any other mod platform. This is because "modrinth", as far as I know, is the only "open source" platform for mods (maybe the minetest platform is too, but it is little known. So what would happen if modrinth supported them a little?).

**Final note:**

I don't know what requirements are needed to carry out this, so I hope the team takes this as a suggestion (a possibility).
something that can be implemented in the future, when the foundations for Minecraft are solid. but who knows, maybe it's easier than I think, and the team just hadn't considered it (optimistic thinking xD)

**Comment by triphora** [comment]
Other games are on [our roadmap](https://docs.modrinth.com/roadmap). We ask that you do not make posts for things that are already planned.

**Reply by 00pandora00** [reply]
I'm sorry I did not know. thanks for the information. I will check that link
